app/models/course/assessment/submission.rb

```diff
@@ -169,6 +169,20 @@ def grade
     latest_answers.map { |a| a.grade || 0 }.sum
   end
 
+  def last_graded_at
```

Comment:
there's a calculated attribute of `last_graded_at`, I think this method is unnecessary.

Reply:
I did not find this attribute?

Reply:
Oops, it's => https://github.com/Coursemology/coursemology2/blob/master/app/models/course/assessment/submission.rb#L77

It's a calculated attribute, one of the gem that we are using: https://github.com/aha-app/calculated_attributes

Reply:
Got it.:P